aten/src/ATen/native/mkldnn/TensorShape.cpp

```diff
@@ -0,0 +1,46 @@
+#include <ATen/ATen.h>
+#include <ATen/Config.h>
+#include <ATen/InferSize.h>
+#include <ATen/NativeFunctions.h>
+
+#if !AT_MKLDNN_ENABLED()
+
+namespace at {
+namespace native {
+
+Tensor mkldnn_view(const Tensor& self, IntArrayRef size) {
+  AT_ERROR("mkldnn_reshape: ATen not compiled with MKLDNN support");
+}
+
+Tensor mkldnn_reshape(const Tensor& self, IntArrayRef size) {
+  AT_ERROR("mkldnn_reshape: ATen not compiled with MKLDNN support");
+}
+
+} // namespace native
+} // namespace at
+
+#else // AT_MKLDNN_EBABLED
+
+#include <ATen/native/mkldnn/MKLDNNCommon.h>
+
+namespace at {
+namespace native {
+
+Tensor mkldnn_view(const Tensor& self, IntArrayRef size) {
```

Comment:
nit: I guess you can move it out of the #ifdef as the error is regardless of whether mkldnn is compiled in

Reply:
The error messages are different though so I would prefer to keep two these two places.